spec/requests/api/visitor_can_see_details_of_an_article_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe Api::ArticlesController, type: :request do
   describe 'GET /article/1 successfully' do
-    let!(:articles) { create(:article, title: 'Article Title', lead: 'At some point there will be something the read in the lead', content: 'Article content will go here for the user to read.') }
+    let!(:articles) { create(:article, title: 'Article Title', lead: 'At some point there will be something the read in the lead', content: 'Article content will go here for the user to read.', category: 'latest_news') }
```

Comment:
it' all about the details ;-) Why is this variable named `articles` and not `article`?

Reply:
Thanks for spotting that! We have now changed so it is consistent :)